shared/chat/conversation/messages/wrapper/chat-send.js

```diff
@@ -75,12 +75,23 @@ class SendIndicator extends React.Component<Props, State> {
     }
   }
 
+  _restart() {
```

Comment:
can you rename this to `_onResend` and move it to right below `_onFailed` above?